api/src/main/resources/workbench-api.yaml

```diff
@@ -3944,6 +3959,32 @@ paths:
           description: A collection of CohortChartData
           schema:
             "$ref": "#/definitions/CohortChartDataListResponse"
+  "/v1/cohortbuilder/{workspaceNamespace}/{workspaceId}/top/chartinfo/{cohortReviewId}/{domain}/{limit}":
+    parameters:
+      - "$ref": "#/parameters/workspaceNamespace"
+      - "$ref": "#/parameters/workspaceId"
+      - "$ref": "#/parameters/cohortReviewId"
+    get:
+      tags:
+        - cohortReview
+      description: Returns a collection of CohortChartData for UI charting in cohort.
+      operationId: getCohortReviewChartData
+      parameters:
+        - in: path
+          name: domain
+          type: string
+          required: true
+          description: represents text domain
+        - in: path
+          name: limit
+          type: integer
+          required: true
+          description: represents top n-items to get
+      responses:
+        200:
+          description: A collection of CohortChartData
+          schema:
+            "$ref": "#/definitions/CohortChartDataListResponse"
```

Comment:
nit: Can we move this into the same section of API calls for cohortReview?

Reply:
done. Also moved one of conceptSets call mixed with dataSet